### DISCUSSION TITLE
Postal_worker 1 keeps stopping

### DISCUSSION THREAD

**Discussion by LazyHatGuy** [postalserver/postal#1574] [Help with using Postal]
I have tried changing the version of RabbitMQ I am running with Docker and I have tried using commands like
```
$ rabbitmqctl set_user_tags YOUR_USERNAME administrator
$ rabbitmqctl set_permissions -p / YOUR_USERNAME ".*" ".*" ".*"
```
With no luck.

I keep getting the following error
```
cron_1      | E, [2021-08-29T15:53:41.184625 #9] ERROR -- #<Bunny::Session:0x55686ba004a0 postal@127.0.0.1:5672, vhost=postal, addresses=[127.0.0.1:5672]>: Authentication with RabbitMQ failed: 403 ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.
cron_1      | [9] [2021-08-29T15:53:41.185] ERROR -- : Authentication with RabbitMQ failed. Please check your connection settings. Username: postal, vhost: postal, password length: 14
```

Any advice?
For reference, I followed this guide: https://docs.postalserver.io/install/installation

**Comment by willpower232** [comment]
You might have to scrap the rabbitmq and start again, making sure to specify the credentials when the container is run for the first time

https://docs.postalserver.io/install/prerequisites#rabbitmq

If you run it without that information and persist the volume then you would have problems like these

**Reply by LazyHatGuy** [reply]
I did it exactly as detailed in that guide but it never worked

**Reply by willpower232** [reply]
have you definitely fully deleted the rabbitmq container and made it again?
